model/ftn/w3fldsmd.ftn

```diff
@@ -1140,7 +1142,7 @@
                     FX0(IX,IY) = FXN(IX,IY)
                     IF (FL2D) FY0(IX,IY) = FYN(IX,IY)
                     END DO
-                  IF( FLST ) THEN
+                  IF( FLST .OR. .NOT.FL2D ) THEN
```

Comment:
No, it will not. The reason is that this line of code (and there is another similar one) is within an if block with the condition:

IF (FLINTERP)

and the FLINTERP is set a few lines above in the same routine. At the moment the only scalar field for which FLINTERP=.TRUE. is the air density.

Reply:
OK - I see that now. Thanks.